titus_isolate/metrics/bpf.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import print_function
+from bcc import BPF, PerfType, PerfHWConfig
```

Comment:
don't forget to add it to `requirements.txt`

Reply:
Sadly can't be added to `requirements.txt`. Must be installed with `apt-get install python3-bcc` on the target host.